registry.json

```diff
@@ -1,4 +1,18 @@
 [
+    {
+    "ticker": "ADP",
+    "poolId": "90454dd6894d47a5cc1e9d16ed4536b876e8e08c3c274d15caa98ff9",
+    "social": {
+      "website": "https://www.adapolis.io/",
+      "twitter": "Adapolis_ADP",
+      "facebook": "",
+      "instagram": "",
+      "youtube": "" 
+      },
+    "telegram": "https://t.me/Adapolis_Stakepools",
```

Comment:
This must be the telegram of an individual and not a group or channel. Also please remove the `https://t.me/`.

Reply:
It's really a must? i will put my privat one, but i think its a mistake... i really don't want that people jump on my privat telegram, that's the reason why exist group for ADP. It's about privacy and security...